src/Entity/EntityTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Acquia\LiftClient\Entity;
+
+use Symfony\Component\Serializer\Encoder\JsonEncoder;
+use Symfony\Component\Serializer\Normalizer\CustomNormalizer;
+use Symfony\Component\Serializer\Serializer;
+
+trait EntityTrait
+{
+    /**
+   * @param string $key
+   * @param string $default
+   *
+   * @return mixed
+   */
+  private function getEntityValue($key, $default)
```

Comment:
This has already changed in the Rule PR, but will also be rearchited since we want to inherit from the entity class, which will make this a base function for all Entity types